### ISSUE TITLE
Neural network training loss

### ISSUE THREAD

**Issue by austinlucaslake** [austinlake04/desiforecast#11]
# Overview 

I have run into a rather unusual situation in which the training loss of network is coming up as negative. The loss function I am using is binary cross entropy with logistic loss, via PyTorch's `torch.nn.BCEwithLogitsLoss()` function. I will be trying to consult with machine learning experts at LBL to determine if they is acceptable.

![Training Loss](https://raw.githubusercontent.com/austinlake04/desiforecast/neural_network/doc/nb/neural_network/neural_network_loss.png)
